### PR TITLE
OSDOCS-3172: Changing terminology from upgrade to update

### PR DESCRIPTION
**Pull request by sagidlow**
Applies to 4.6+
JIRA: https://issues.redhat.com/browse/OSDOCS-3172. Changes based off this: https://github.com/openshift/openshift-docs/pull/40525
SME ack required.
QE ack not required.
Preview Link: 

- [Understanding OpenShift updates](https://deploy-preview-43138--osdocs.netlify.app/openshift-enterprise/latest/updating/understanding-openshift-updates.html)
- [Understanding upgrade channels](https://deploy-preview-43138--osdocs.netlify.app/openshift-enterprise/latest/updating/understanding-upgrade-channels-release.html)
- [Updating a restricted network cluster](https://deploy-preview-43138--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-restricted-network-cluster.html)
- [Updating a cluster using the CLI](https://deploy-preview-43138--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster-cli.html)
- [Updating a cluster using the web console](https://deploy-preview-43138--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster-within-minor.html)
- [Updating a cluster that includes RHEL compute machines](https://deploy-preview-43138--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster-rhel-compute.html)
- [Updating hardware on nodes running on vSphere](https://deploy-preview-43138--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-hardware-on-nodes-running-on-vsphere.html)

### CODE REVIEW

**Comment by sagidlow** [comment]
@LalatenduMohanty / @sdodson If either of you could take a look. TY!

**Comment by sdodson** [comment]
/lgtm
Thanks for updating things to be uniform.

File: updating/updating-cluster-cli.adoc [hunk 1]
```diff
@@ -17,13 +17,13 @@ You can update, or upgrade, an {product-title} cluster within a minor version by
...
-* Ensure that you address all `Upgradeable=False` conditions so the cluster allows an upgrade to the next minor version. You can run the `oc adm upgrade` command for an output of all `Upgradeable=False` conditions and the condition reasoning to help you prepare for a minor version upgrade.
+* If your cluster uses manually maintained credentials with the AWS Secure Token Service (STS), obtain a copy of the `ccoctl` utility from the release image being updated to and use it to process any updated credentials. For more information, see xref:../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#sts-mode-upgrading[Upgrading an OpenShift Container Platform cluster configured for manual mode with STS].
+* Ensure that you address all `Upgradeable=False` conditions so the cluster allows an update to the next minor version. An alert displays at the top of the *Cluster Settings* page when you have one or more cluster Operators that cannot be upgraded. You can still update to the next available patch update for the minor release you are currently on.
```
Comment (mburke5678) [inline updating/updating-cluster-cli.adoc:26]: > Operators that cannot be upgraded
Operators get upgraded, not updated?

File: updating/updating-cluster-within-minor.adoc [hunk 2]
```diff
@@ -17,12 +17,12 @@ Use the web console or `oc adm upgrade channel _<channel>_` to change the update
...
-* If your cluster uses manually maintained credentials, ensure that the Cloud Credential Operator (CCO) is in an upgradeable state. For more information, see xref:../updating/updating-cluster-within-minor.adoc#manually-maintained-credentials-upgrade_updating-cluster-within-minor[Upgrading clusters with manually maintained credentials].
-* If your cluster uses manually maintained credentials with the AWS Secure Token Service (STS), obtain a copy of the `ccoctl` utility from the release image being upgraded to and use it to process any updated credentials. For more information, see xref:../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#sts-mode-upgrading[Upgrading an OpenShift Container Platform cluster configured for manual mode with STS].
+* If your cluster uses manually maintained credentials, ensure that the Cloud Credential Operator (CCO) is in an upgradeable state. For more information, see xref:../updating/updating-cluster-cli.adoc#manually-maintained-credentials-upgrade_updating-cluster-cli[Upgrading clusters with manually maintained credentials].
```
Comment (mburke5678) [inline updating/updating-cluster-within-minor.adoc:24]: Why did you change the link? The current link works fine. These changes take you to the same module, but in a different assembly.

**Comment by mburke5678** [comment]
@sagidlow A couple of nits. Otherwise LGTM

**Review by mburke5678 — APPROVED** [review]

File: updating/updating-cluster-cli.adoc (hunk 1 above)
Comment (sagidlow) [reply updating/updating-cluster-cli.adoc:26]: Correct, Operators get upgraded.

Comment (bmcelvee) [inline updating/updating-cluster-cli.adoc:26]: Sneaky "upgraded". Should this be changed?

s/upgraded/updated

Comment (sagidlow) [reply updating/updating-cluster-cli.adoc:26]: Operators are upgraded, and clusters are updated. Leaving the content as is.
